saleor/graphql/webhook/subscription_types.py

```diff
@@ -465,6 +465,16 @@ class Meta:
         )
 
 
+class OrderExpired(SubscriptionObjectType, OrderBase):
+    class Meta:
+        root_type = "Order"
+        enable_dry_run = True
+        interfaces = (Event,)
+        description = (
+            "Event sent when order is expired." + ADDED_IN_312 + PREVIEW_FEATURE
```

Comment:
Maybe `Event sent when order becomes expired`? Seems a bit clearer to me, when I was reading that I wondered if this is used in a conditional while checking some orders(?), but that's only triggered when a task marks it as such under some conditions automatically. wdyt?

Reply:
Sure, that's sounds better.